cnames_active.js

```diff
@@ -3259,8 +3259,7 @@ var cnames_active = {
   "tgsnake": "cname.vercel-dns.com", // noCF
   "tgx-core": "tgx-core.github.io/website",
   "thea": "idokutela.github.io/thea",
-  "thegrid": "antoniandre.github.io/grid",
-  "theme-next": "hexo-theme-next.netlify.app",
+  "thema": "themakit.github.io",
```

Comment:
```suggestion
  "thegrid": "antoniandre.github.io/grid",
  "thema": "themakit.github.io",
  "theme-next": "hexo-theme-next.netlify.app",
```
Please do not remove existing records